### DISCUSSION TITLE
Erro a atualizar

### DISCUSSION THREAD

**Discussion by andremp15** [RamonSilva20/mapos#2495] [General]
Olá pessoal está acontecendo um erro no programa quando eu tento fazer atualização tanto no servidor em máquina próprio quanto em servidor contratado no meu caso a hostgator sempre dá esse erro, e cria duas pastas dentro do diretório conforme as imagens mostra abaixo e o programa não atualiza. A versão que eu estou utilizando para testar é a versão mais recente 4.45

![IMG-20240718-WA0019.jpg](https://github.com/user-attachments/assets/5e9af6cd-dda2-4f72-abd9-7d0cde020a81)

![IMG-20240718-WA0018.jpg](https://github.com/user-attachments/assets/231aaf28-3560-43c3-908a-1aa4db9060a5)

**Comment by sigmaonbr** [comment]
Olá estou tendo o mesmo problema e uso uma hospedagem de minha revenda.

**Comment by johnteckinformatica** [comment]
Fui atualizar 4.44 para 4.47 também deu erro.
Se fizer a instalação limpa funciona, mas quando restaura o backup, o sistema morre.

![image](https://github.com/user-attachments/assets/560e0107-efe3-433b-9698-637dd4e74b9a)

**Reply by andremp15** [reply]
Boa noite eu consegui resolver o problema manualmente editando o backup do SQL na versão 4.42 que eu estava utilizando, copiei e colei os produtos os clientes e etc para a versão mais recente 4.47, deu certo para mim, e não deu mais erro

Porém tive que fazer outras alterações o arquivo imprimirOS e também os botões não gostei da versão 447 e os botão em cima do outro ficou muito feio, mas em compensação o novo modelo da impressão da OS está muito top.

**Comment by lucasbmira1** [comment]
A atualização realmente é complicada, eu também tive problemas, não consegui!!

**Comment by johnteckinformatica** [comment]
Já consegui fazer no meu, tive que exportar as tabelas que utilizo pelo phpmyadmin.
(como andremp15 deu a dica)

Como uso só o básico, foi até de boa. Não uso a parte de faturamento, vendas.
Ai exportei só: anotacoes, clientes, os, produtos, produtos_os, servicos, servicos_os.

Ai criei um banco de dados novo, importei primeiro: anotacoes, clientes, produtos, servicos -  depois: produtos_os e servicos_os

Depois você põe a instalação nova do mapos e faz a configuração inicial, pra ele recriar o restante do banco de dados.

Ai vai precisar configurar o emitente, financeiro pix, email caso use.

**Comment by MarceloGrass** [comment]
Muito obrigado amigos  >(andremp15 e johnteckinformatica), estava tentando atualizar da versão 4.39 e conseguia instalar do zero na Hostinger com o tutorial no YouTube de (edson edson) excelente, mas na hora de importar os dados da base antiga dava erro mesmo usando os sql de updates, mas importando os dados primeiro a base vazia e depois instalando em cima e depois se importando o sql de upgrade deu certo. E substitui as imagens de logo etc na pasta assets. Fico show!!!
